addons/web/controllers/main.py

```diff
@@ -424,6 +424,20 @@ def xml2json_from_elementtree(el, preserve_whitespaces=False):
     res["children"] = kids
     return res
 
+def _password_warn(env, partner):
+    """ Admin still has `admin` password, flash a message via chatter
+
+    Does not use message_post because depending on conf' that might send an
```

Comment:
Is it me or you're actually using `message_post` now? ;-)

Reply:
Kinda yeah, that was about using mail thread's message_post directly.